packages/client/hmi-client/src/components/model/petrinet/tera-petrinet-parts.vue

```diff
@@ -3,60 +3,74 @@
 		<AccordionTab>
 			<template #header>
 				State variables<span class="artifact-amount">({{ states.length }})</span>
+				<tera-input-text class="ml-auto" placeholder="Filter" v-model="statesFilter" />
 			</template>
 			<tera-states
 				v-if="!isEmpty(mmt.initials)"
 				:model="model"
 				:mmt="mmt"
 				:feature-config="featureConfig"
+				:filter="statesFilter"
 				@update-state="emit('update-state', $event)"
 			/>
 		</AccordionTab>
 		<AccordionTab>
 			<template #header>
 				Parameters<span class="artifact-amount">({{ parameters.length }})</span>
+				<tera-input-text class="ml-auto" placeholder="Filter" v-model="parametersFilter" />
 			</template>
 			<tera-parameters
 				v-if="!isEmpty(mmt.parameters)"
 				:model="model"
 				:mmt="mmt"
 				:mmt-params="mmtParams"
 				:feature-config="featureConfig"
+				:filter="parametersFilter"
 				@update-parameter="emit('update-parameter', $event)"
 			/>
 		</AccordionTab>
 		<AccordionTab>
 			<template #header>
 				Observables <span class="artifact-amount">({{ observables.length }})</span>
+				<tera-input-text class="ml-auto" placeholder="Filter" v-model="observablesFilter" />
 			</template>
 			<tera-observables
 				v-if="!isEmpty(observables)"
 				:model="model"
 				:mmt="mmt"
 				:observables="observables"
 				:feature-config="featureConfig"
+				:filter="observablesFilter"
 				@update-item="emit('update-observable', $event)"
 			/>
 		</AccordionTab>
 		<AccordionTab>
 			<template #header>
 				Transitions<span class="artifact-amount">({{ transitions.length }})</span>
+				<tera-input-text class="ml-auto" placeholder="Filter" v-model="transitionsFilter" />
 			</template>
 			<tera-transitions
 				v-if="!isEmpty(transitions) && !isEmpty(mmt.templates)"
 				:mmt="mmt"
 				:mmt-params="mmtParams"
 				:transitions="transitions"
 				:feature-config="featureConfig"
+				:filter="transitionsFilter"
 				@update-transition="emit('update-transition', $event)"
 			/>
 		</AccordionTab>
 		<AccordionTab>
 			<template #header>
-				Time
-				<span class="artifact-amount">({{ time.length }})</span>
+				Time <span class="artifact-amount">({{ time.length }})</span>
+				<tera-input-text class="ml-auto" placeholder="Filter" v-model="timeFilter" />
```

Comment:
I don't think we need a filter here - a situation where we have many time parameters is more like an error-case. Let's remove this.